PowerNSX.psm1

```diff
@@ -26788,6 +26913,18 @@ function Add-NsxLoadBalancerPoolMember {
     This cmdlet adds a new member to the specified LoadBalancer Pool and
     returns the updated Pool.
 
+    .EXAMPLE
+    Adds a Pool Member to Load Balanver Pool WebPool01 on Edge01
```

Comment:
Balanver -> Balancer ?

Avoid to add text before example because powershell think it is a code... (and you get PS />Adds A pool
when use --examples